data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/v2/beacon/GetBlockV2IntegrationTest.java

```diff
@@ -82,10 +83,25 @@ public void shouldGetAltairBlockAsSsz() throws IOException {
     assertThat(response.header(HEADER_CONSENSUS_VERSION)).isEqualTo(Version.altair.name());
   }
 
+  @Test
+  public void shouldGetAltairBlockAsGzip() throws IOException {
+    startRestAPIAtGenesis(SpecMilestone.ALTAIR);
+    createBlocksAtSlots(10);
+    final Response response = getGzip("head", OCTET_STREAM);
```

Comment:
My suggestion is to get the response as JSON just to make deserializing it into an object a bit easier. This way all we need to do is decompress the Gzip into a string and parse it into a `GetBlockResponseV2` object.

Not that it can't be done with SSZ. So it is up to you :)